### DISCUSSION TITLE
Floating short cut panel on mobile

### DISCUSSION THREAD

**Discussion by paulyili** [outline/outline#7445] [Self Hosting]
I'm not sure how to describe the issue. But on mobile when using the on screen keyoboad, the shortcut tab with h1 h2 etc floats around the screen rather than attached to the keyboard. 

If I scroll, it sits in the middle of the screen. Makes using the app on mobile quite annoying.  

Can I disable it ? Workarounds ?

**Comment by tommoor** [comment]
Is this Android?

**Reply by paulyili** [reply]
Yep it is.
